circuits/integration/zk_circuits/valid_match_mpc.rs

```diff
@@ -450,14 +415,15 @@ fn test_valid_match__same_side(test_args: &IntegrationTestArgs) -> Result<(), St
 /// Test the case in which the balance provided to the matching engine is not for
 /// the correct asset
 fn test_valid_match__invalid_balance_mint(test_args: &IntegrationTestArgs) -> Result<(), String> {
-    let party_id = test_args.party_id;
+    let fabric = &test_args.mpc_fabric;
+    let party_id = fabric.party_id();
     let price = *DUMMY_PRICE;
     let my_order = create_test_order(party_id);
     let mut my_balance = create_test_balance(party_id);
     let amount = compute_max_amount(price, &my_order, &my_balance);
 
     // Switch the mint of the balance to be the wrong asset in the pair
-    if test_args.party_id == 0 {
+    if fabric.party_id() == PARTY0 {
```

Comment:
nit: here (and in the other tests), can we just use the `party_id` variable declared at the top of the test function instead of `fabric.party_id()`? not sure if it's expected to change